lib/ansible/cli/config.py

```diff
@@ -141,6 +145,14 @@ def run(self):
         elif context.CLIARGS['action'] == 'view':
             raise AnsibleError('Invalid or no config file was supplied')
 
+        # Galaxy server config is not defined as part of the base.yml config or
+        # a plugin definition. The defs should be loaded if this type was
+        # requested.
```

Comment:
i would say it is part of 'base' though dynamically generated vs from the static base.yml